http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/HttpSseIT.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.ts.http.advanced;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusScenario
+@Tag("https://github.com/quarkusio/quarkus/issues/36402")
+// this test should only fail on native
+public class HttpSseIT {
+
+    @QuarkusApplication
+    static RestService app = new RestService()
+            .withProperty("quarkus.oidc.enabled", "false")
+            .withProperty("quarkus.keycloak.policy-enforcer.enable", "false")
+            .withProperty("quarkus.keycloak.devservices.enabled", "false");
+    private static final String SSE_ERROR_MESSAGE = "java.lang.ClassNotFoundException: Provider for jakarta.ws.rs.sse.SseEventSource.Builder cannot be found";
+
+    @Test
```

Comment:
While I think it's nicely clear that you want to test `sse` as you have it in dedicated class, I'd still prefer if you added it to the `HttpAdvancedIT` because that way, application won't be re-build. There are miscellaneous test and if you choose fitting unit test method name, it will be clear enough. We can spare couple of minutes because `quarkus.oidc.enabled` is a build time property.

It's not irrational, because if you look how long our native CI is running, we need to have good reasons to add it as a separate class.

Reply:
Now that I'm thinking about it, you probably wanted to avoid OCP tests. Well, I think it is positive to have this tested on OCP as well.

Reply:
Moved SSE test to HttpAdvancedIT.